framework/wazuh/core/indexer/events.py

```diff
@@ -1,29 +1,57 @@
+import asyncio
 from dataclasses import asdict
+from typing import List
+from uuid import UUID
 
 from .base import BaseIndex
 from wazuh.core.indexer.models.events import Events
+from wazuh.core.indexer.bulk import MixinBulk
 
+from opensearchpy import AsyncOpenSearch
 
-class EventsIndex(BaseIndex):
+
+HARDCODED_EVENTS_INDEX_NAME = "events"
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/wazuh/wazuh/commit/47e4508357e0fb53f499847d6a791a0f4c86f15b.